blackbox/docs/sql/system.txt

```diff
@@ -1040,23 +1032,26 @@ system column in the query::
     +---------------+--------...-+-----...-+------------+
     SELECT 1 row in set (... sec)
 
+An operation is a node-specific sub-component of a job (for when a job involves
+multi-node processing). Jobs that do not require multi-node processing will not
+produce any operations.
+
 .. note::
 
-    In some cases internal "sub-jobs" are created. Parts of these sub-jobs may
-    appear in `sys.operations`. These entries don't have corresponding entries
-    in `sys.jobs`.
+    In some cases, operations are generated for internal CrateDB work that does
+    not directly correspond to a user request. These entries do not have
+    corresponding entries in ``sys.jobs``.
 
 .. _sys-logs:
 
 Logs
 ----
 
-Both ``sys.jobs`` and ``sys.operations`` have a corresponding ``_log`` table.
+The ``sys.jobs`` and ``sys.operations`` tables have corresponding log tables:
+``sys.jobs_log`` and ``sys.operations_log``.
 
-After a job or operation finishes the entries will be moved into the ``_log``
-table. The size of ``_log`` tables is determined by a fixed size
-(:ref:`stats.jobs_log_size <stats.jobs_log_size>`) or by an expiration time
-(:ref:`stats.jobs_log_expiration <stats.jobs_log_expiration>`)::
+After a job or operation finishes, the corresponding entry will be moved into
+the corresponding log table::
 
```

Comment:
@nslater yes, sounds confusing. However, only the `sys.jobs_log` table contains also invalid queries. `sys.jobs` instead gives only information about the currently executed queries. Since invalid queries are not event executed, they are not listed.